File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int> fre;
        for(int i=0;i<nums.size();i++){
            fre[nums[i]]++;
        }

        for(auto it:fre){
            if(it.second>1){
                return true;
            }
        }
        return false;
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        fre = {}
        for n in nums:
            if n in fre:
                fre[n] += 1
            else:
                fre[n] = 1
    
        if all(v == 1 for v in fre.values()):
            return False
        else:
            return True